File: app/utils/llms.py
from langchain_openai import ChatOpenAI
import os

OpenAI_GPT4O = ChatOpenAI(
        model="gpt-4o", 
        api_key=os.getenv("OPENAI_API_KEY"), 
        max_tokens=8192,
        temperature=0.3
    )

File: node/llms.ts
import { ChatAnthropic } from "@langchain/anthropic";
import { ChatOpenAI } from "@langchain/openai";
import { BaseChatModel } from "@langchain/core/language_models/chat_models";

export const Anthropic_3_5_Sonnet: BaseChatModel = new ChatAnthropic({
  model: "claude-3-5-sonnet-20241022",
  temperature: 0.3,
  apiKey: process.env.ANTHROPIC_API_KEY,
});

export const OpenAI_GPT_4o: BaseChatModel = new ChatOpenAI({
  model: "gpt-4o",
  temperature: 0.3,
  apiKey: process.env.OPENAI_API_KEY,
});

export const OpenAI_GPT_4o_Mini: BaseChatModel = new ChatOpenAI({
  model: "gpt-4o-mini",
  temperature: 0.3,
});
